test-plans/README.md

```diff
@@ -0,0 +1,33 @@
+# test plans
+
+Contains all the test-plans of all our services, and the test plan templates.
+
+## folder structure
+
+the test-plans belong to their corresponding folder. For Example a test plan created for the Moorhuhn frontend:
+
+```
+test-plans/minigames/moorhuhn/frontend/plans
+```
+
+As soon as a tester has tested a feature according to a test plan, it has to be placed in the `executed` folder
+
+## naming convention
+
+test-plan files should be following this structure
+
+```
+name-service-backend/frontend.md
+example:
+highscore-moorhuhn-frontend.md
+```
+
+where `name` is the name of the feature the test plan covers
```

Comment:
```suggestion
where `$ACTION` is the name of the feature the test plan covers
```
↑ Also, as I understood it: There should basically be one test plan per service, each having multiple test cases.